go.mod

```diff
@@ -4,11 +4,11 @@ go 1.16
 
 require (
 	github.com/google/go-cmp v0.5.6
-	github.com/projectcontour/contour v1.19.1
+	github.com/projectcontour/contour v1.20.1
 	go.uber.org/zap v1.19.1
-	k8s.io/api v0.22.5
-	k8s.io/apimachinery v0.22.5
-	k8s.io/client-go v0.22.5
+	k8s.io/api v0.23.1
+	k8s.io/apimachinery v0.23.1
+	k8s.io/client-go v0.23.1
```

Comment:
Curious if this k8s bump was necessary?

Reply:
Technically it's fine since our next release (1.4) targets K8s 1.22 (which the k8s 0.23 libs support)

Reply:
Oh! I didn't mean to do that, I updated the Contour version in `./hack/update-deps.sh` and ran it. Is there any way to not include those changes but to update Contour?

Reply:
Bumping contour will bump that - since that version becomes the 'min version'.

https://github.com/projectcontour/contour/blob/002720c894dde8f7a397e97ccbd221752f4374de/go.mod#L32-L34

You could use go `replace` directives but it'll probably be a mess in other ways. So I say let's hold this PR and get the K8s bump in first and rebase after that.